src/ontology/Makefile

```diff
@@ -256,6 +255,9 @@ $(ONT).json: $(ONT).owl
 	$(ROBOT) convert --input $< --check false -f json -o $@.tmp.json &&\
 		mv $@.tmp.json $@
 
+$(ONT)_nodes.tsv $(ONT)_edges.tsv: $(ONT).json
+	kgx transform --input-format obojson --output-format tsv --output $(ONT) $< && test -f $(ONT)_nodes.tsv
```

Comment:
Is a check that `$(ONT)_edges.tsv` exists also needed?

Reply:
I dont think so, if `kgx transform` produces the nodes file, it will produce the edges file.